api/tests/routes/backoffice/accounts_test.py

```diff
@@ -1280,7 +1291,7 @@ def test_update_all_fields(self, legit_user, authenticated_client):
         assert user.firstName == form_data["first_name"].strip()
         assert user.lastName == form_data["last_name"].strip()
         assert user.email == form_data["email"]
-        assert user.dateOfBirth == date_of_birth
+        assert user.dateOfBirth != form_data["birth_date"]
```

Comment:
Attention, ici tu compares un `datetime.datetime` avec un `datetime.date`. Donc même si ça avait écrasé avec la valeur de `form_data`, ce serait probablement quand même `!=`. La vérification n'est pas forcément pertinente. Peut-être à comparer ainsi :

```suggestion
        assert user.dateOfBirth.date() != form_data["birth_date"]
```